src/main/kotlin/net/atos/zac/zoeken/model/zoekobject/DocumentZoekObject.kt

```diff
@@ -0,0 +1,171 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.zoeken.model.zoekobject
+
+import net.atos.client.zgw.drc.model.generated.StatusEnum
+import net.atos.zac.zoeken.model.DocumentIndicatie
+import nl.lifely.zac.util.NoArgConstructor
+import org.apache.solr.client.solrj.beans.Field
+import java.util.Date
+import java.util.EnumSet
+import java.util.Locale
+import java.util.function.Supplier
+import java.util.stream.Collectors
+
+@NoArgConstructor // required for Java bean inspection
+data class DocumentZoekObject(
+    @Field("id")
+    var uuid: String,
```

Comment:
Why not use `objectId` here?

Reply:
Or just `id` everywhere. Beats me why they thought of 3 different names for the same variable indeed..

Reply:
fixed this to use id everywhere. still kept the `getId()` interface method